tests/UI/specs/SiteSelector_spec.js

```diff
@@ -26,6 +26,7 @@ describe("SiteSelector", function () {
 
         await page.waitForSelector('.custom_select_ul_list');
         await page.waitForNetworkIdle();
+        await page.click('.websiteSearch');
```

Comment:
The focused element was changing randomly. This should ensure it's always the same, so the tests won't fail randomly anymore.